packages/elements/src/tab-bar/index.ts

```diff
@@ -22,7 +22,7 @@ const BAR_TRAVEL_DISTANCE = 150; // scroll distance
 /**
  * Container for tabs
  *
- * @fires value-changed - Fired when the `value` changes.
+ * @fires value-changed - Fired at controls when the user changed an active tab. The event is not triggered if `value` property is changed by programmatically.
```

Comment:
Fired ~~at controls~~ when the user change"s"~~d~~ an active tab. The event is not triggered if `value` property is changed ~~by~~ programmatically.